src/java.base/share/conf/security/java.security

```diff
@@ -211,6 +211,10 @@ RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.tls.disabledAlgorithms = \
     TLS_DHE_DSS_WITH_AES_256_GCM_SHA384, \
     TLS_DHE_RSA_WITH_AES_128_CBC_SHA, \
     TLS_DHE_RSA_WITH_AES_256_CBC_SHA, \
+    TLS_DHE_RSA_WITH_AES_128_CBC_SHA256, \
+    TLS_DHE_RSA_WITH_AES_128_GCM_SHA256, \
+    TLS_DHE_RSA_WITH_AES_256_CBC_SHA256, \
+    TLS_DHE_RSA_WITH_AES_256_GCM_SHA384, \
```

Comment:
Please add new entries in sorted order.
While you're here, please correct lines 200,201 which are reversed (EC should be mentioned before ECDH).

Reply:
Those 4 lines are in sorted order. Also, line 200(EC) is before 201(ECDH).

Reply:
I'm not sure how I thought lines 200 and 201 were reversed; sorry for that distraction.

However, while lines 214-217 are themselves in order, they're not all added where they belong. The two "128" additions should precede the existing "256" line:
```
    TLS_DHE_RSA_WITH_AES_128_CBC_SHA, \
    TLS_DHE_RSA_WITH_AES_128_CBC_SHA256, \
    TLS_DHE_RSA_WITH_AES_128_GCM_SHA256, \
    TLS_DHE_RSA_WITH_AES_256_CBC_SHA, \
    TLS_DHE_RSA_WITH_AES_256_CBC_SHA256, \
    TLS_DHE_RSA_WITH_AES_256_GCM_SHA384, \
```

Reply:
updated